### PR TITLE
Add support for a contest-wide scoreboard warning message

### PR DESCRIPTION
**Pull request by thijskh**
Useful when specific contests are used for testing and to avoid any confusion about their officialdom.

### CODE REVIEW

**Comment by thijskh** [comment]
@meisterT I moved the message down a few lines in the template vs your commit dd381242598462b911826cf98cda57d842b2385c to avoid the extra if for current-contest

**Review by meisterT — APPROVED** [review]

**Comment by meisterT** [comment]
We probably want to add an attribute to `contest.yaml` and import it via the API if set.

File: webapp/src/Service/ImportExportService.php [hunk 1]
```diff
@@ -66,6 +66,9 @@ public function getContestYamlData(Contest $contest): array
             'start-time' => Utils::absTime($contest->getStarttime(), true),
             'duration' => Utils::relTime($contest->getContestTime((float)$contest->getEndtime())),
         ];
+        if ($warnMsg = $contest->getWarningMessage()) {
+            $data['warningMessage'] = $warnMsg;
```
Comment (meisterT) [inline webapp/src/Service/ImportExportService.php:70]: nit: `warning-message`? so it matches the other attributes better?

**Comment by meisterT** [comment]
If you rebase on main the unit test will pass.

**Comment by thijskh** [comment]
Already done

**Comment by thijskh** [comment]
I can see the advantage. If we’d do that we need some way to keep it up to date/can we install it with composer?

Of course adding this by default adds a high shoot in foot factor and also allows admins to delete their own auditlog lines. Something to consider. Maybe a feature toggle can help

**Comment by meisterT** [comment]
Is this the right PR for this comment?

**Pull request merged**